Functions.Templates/Templates/CosmosDBTrigger-CSharp/CosmosDBTriggerCSharp.cs

```diff
@@ -0,0 +1,32 @@
+#if (portalTemplates)
+#r "Microsoft.Azure.Documents.Client"
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.Documents;
+
+public static void Run(IReadOnlyList<Document> input, TraceWriter log)
+#endif
+#if (vsTemplates)
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.WebJobs.Extensions.DocumentDB;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Host.Triggers;
+
+namespace Company.Function
+{
+    public static class CosmosDBTriggerCSharp
+    {
+        [FunctionName("CosmosDBTriggerCSharp")]
```

Comment:
The value of FunctionName and the file name should be same as the value of sourceName property in the template.json